### PR TITLE
Automatically trigger weekly deployments on fleek

### PR DESCRIPTION
**Pull request by zebateira**
This will make sure that events stay up to date and mailchimp subscriptions are triggered.
@terichadbourne let me know if weekly cadence is enough. It's set to trigger every Monday at 9am UTC.

### CODE REVIEW

**Comment by zebateira** [comment]
update to trigger everyday at 9am UTC

File: .github/workflows/weekly-build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Weekly Build (events and mailchimp subscriptions updates)
```
Comment (terichadbourne) [inline .github/workflows/weekly-build.yml:1]:
```suggestion
name: Daily Build (events and Mailchimp subscription updates)
```

**Review by terichadbourne — CHANGES_REQUESTED** [review]
Would suggest updating the name to match the new schedule we discussed today. Anything you need me to do to test this one? If not, please go ahead and merge when ready!

File: .github/workflows/weekly-build.yml (hunk 1 above)
Comment (zebateira) [reply .github/workflows/weekly-build.yml:1]: this is it! will merge

**Pull request merged**